vllm/v1/sample/sampler.py

```diff
@@ -93,6 +93,10 @@ def sample(
             sampling_metadata.no_top_p,
             sampling_metadata.top_p,
         )
+
+        if not sampling_metadata.no_min_p:
+            logits = self.apply_min_p(logits, sampling_metadata.min_p)
```

Comment:
@AoyuQC @WoosukKwon sorry I didn't look at this close enough before and just noticed this is wrong. It needs to be done before sampling of course or has no effect :)

I have included a fix in https://github.com/vllm-project/vllm/pull/13311

Reply:
Oh you're right 😓 I didn't take a close look. Thanks for catching it!